assets/stylesheets/common/chat-channel-info.scss

```
@@ -119,7 +119,7 @@ input.channel-members-view__search-input {
 .channel-members-view__list-item {
   display: flex;
   align-items: center;
-  padding: 0.5rem 0;
+  padding: 0.5rem 0 0 1px;
```

```suggestion
  padding: 0.5rem 0 0.5rem 1px;
```

I think you should change like this to achieve the previous one.